src/workflow/ArcanistDiffWorkflow.php

```
@@ -419,6 +427,9 @@ public function run() {
         'message' => $commit_message,
       ));
 
+    // TODO ProdSec: Handle exit code and stderr after monitoring phase is complete
```

failing silently feels risky, can we handle now?

---

We are planning on adding alerts to errors which will be handled on security end.